projects/dotcms-js/src/lib/core/site.service.ts

```diff
@@ -147,13 +148,11 @@ export class SiteService {
      */
     switchSite(site: Site): void {
         this.loggerService.debug('Applying a Site Switch', site.identifier);
-
         this.coreWebService
             .requestView({
                 method: RequestMethod.Put,
                 url: `${this.urls.switchSiteUrl}/${site.identifier}`
-            })
-            .subscribe(() => this.setCurrentSite(site));
```

Comment:
Don't we need subscribe here?

Reply:
no, because we now hear the `SWITCH_SITE ` event. https://github.com/dotCMS/core-web/pull/1268/files#diff-7a9328737943182522e1d2e3d7165d0dR52